<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAddressDialog</class>
 <widget class="QDialog" name="NewAddressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>584</width>
    <height>255</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>584</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New contact</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_labelTextLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>LABEL</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="WalletGui::WalletCancelButton" name="m_cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="WalletGui::ClearableLineEdit" name="m_labelEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">[errorState=&quot;true&quot;] {
  border-color: #ef3131;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="WalletGui::WalletOkButton" name="m_okButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_addressTextLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>ADDRESS</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>338</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="WalletGui::ClearableLineEdit" name="m_addressEdit">
     <property name="styleSheet">
      <string notr="true">[errorState=&quot;true&quot;] {
  border-color: #ef3131;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="WalletGui::WalletGrayCheckBox" name="m_donationCheck">
       <property name="text">
        <string>DONATION ADDRESS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WalletGui::WalletSmallGrayTextLabel" name="m_donationWarningLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Donations to this address will be stopped.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletGui::WalletSmallGrayTextLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/WalletTextLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletOkButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Common/WalletOkButton.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletCancelButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Common/WalletCancelButton.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletNormalGrayTextLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/WalletTextLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Gui/Common/ClearableLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletGrayCheckBox</class>
   <extends>QCheckBox</extends>
   <header>Gui/Common/WalletGrayCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_labelEdit</tabstop>
  <tabstop>m_addressEdit</tabstop>
  <tabstop>m_okButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>NewAddressDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NewAddressDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewAddressDialog</receiver>
   <slot>validateAddress(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_labelEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewAddressDialog</receiver>
   <slot>validateLabel(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_donationCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>NewAddressDialog</receiver>
   <slot>donationCheckStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validateAddress(QString)</slot>
  <slot>validateLabel(QString)</slot>
  <slot>donationCheckStateChanged(int)</slot>
 </slots>
</ui>
